weblate/billing/models.py

```diff
@@ -249,7 +252,7 @@ def all_projects(self):
 
     @cached_property
     def projects_display(self):
-        return ", ".join(str(x) for x in self.all_projects)
+        return format_html_join_comma("{}", ((str(x),) for x in self.all_projects))
```

Comment:
`str` is not needed here, format does it anyway:

```suggestion
        return format_html_join_comma("{}", ((x,) for x in self.all_projects))
```